PHPCompatibility/Tests/BaseSniffTestCase.php

```diff
@@ -377,4 +377,45 @@ private function gatherIssues($issuesArray)
 
         return $allIssues;
     }
+
+    /**
+     * Run assertions that the fixer output matches the ".fixed" file for a particular sniff.
+     *
+     * @since 10.0.0
+     *
+     * @param string $pathToTestFile   Absolute path to the file to sniff.
+     *                                 Allows for passing __FILE__ from the unit test
+     *                                 file. In that case, the test case file is presumed
+     *                                 to have the same name, but with an `inc` extension.
+     * @param string $targetPhpVersion Value of 'testVersion' to set on PHPCS object.
+     *
+     * @return void
+     */
+    protected function assertFixerOutputMatches($pathToTestFile, $targetPhpVersion = 'none')
```

Comment:
This setup does not allow for testing that fixes may be different depending on the `$targetPhpVersion`, nor allow for a `$targetPhpVersion` in which no fixes would be made.